src/style.css

```diff
@@ -1,3 +1,80 @@
 body {
+  font-family: Roboto, Arial, sans-serif;
   margin: 0;
+  padding: 0;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  background-color: #fff;
+  padding: 0 50px;
+  max-width: 1200px;
+}
+
+.go-home {
+  margin: 10px 0;
+  height: 40px;
+}
+
+.logo {
+  height: 40px;
+  width: 40px;
+}
+
+.nav {
+  display: flex;
+}
+
+.nav__list {
+  display: flex;
+  align-items: center;
+  list-style: none;
+  text-transform: uppercase;
+  margin: 0;
+  padding: 0;
+}
+
+.nav__item {
+  position: relative;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100%;
```

Comment:
you don't need this `height: 100%;`

Reply:
If I remove this height, clickable area disappears and the underline of the active page rises to the text.

Reply:
> you don't need this `height: 100%;`

If I remove this height, clickable area disappears and the underline of the active page rises to the text.